frontend/src/pages/LandingPage/styles.ts

```diff
@@ -0,0 +1,125 @@
+import styled from 'styled-components';
+
+const LandingPageContainer = styled.div`
+  margin-top: 30px;
+`;
+const LandingPageTitle = styled.h1`
+  font-size: 50px;
+  background: -webkit-linear-gradient(30deg, #9f62de, #b77eff);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  text-align: center;
+  position: relative;
+`;
+
+const CardContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: space-evenly;
+  position: relative;
+`;
+const Card = styled.div`
+  background: #fff;
+  border-radius: 20px;
+  display: inline-block;
+  height: 275px;
+  margin: 1rem;
+  position: relative;
+  width: 250px;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
+  padding: 45px 25px;
+  transition: all 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
+  &:hover {
+    box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
+  }
+`;
+
+const iconContainer = styled.div`
```

Comment:
captialise first letter of styled component variable (along with the below declaration aswell)

Reply:
y